src/lib/analyzer.svelte

```diff
@@ -64,56 +64,118 @@
 
    var fields = [
      "Manufacturer Code",
-     "Device Class",
-     "Device Function"
+     "Device Class"
    ]
    
+   var prodinfofields = [
+     "Model ID",
+     "Software Version Code"
+   ]
+
+   var desc  = "";
+   var swver = "";
+
    for (var k in lst) {
      var m = lst[k];
      if (m.pgn == 60928) {
-       var d = "";
        for (var kk in fields) {
          var vv = fields[kk];
          if (vv in m) {
-           d += m[vv] + " ";
+           desc += m[vv] + " ";
+         }
+       }
+     }
+     if (m.pgn == 126996) {
+       for (var kk in prodinfofields) {
+         var vv = prodinfofields[kk];
+         if (vv in m) {
+           swver += m[vv] + " ";
          }
        }
-       return d;
      }
    }
-   return "no desc";
+   if (desc == "") {
+     return "No description";
+   }
+   desc = desc + swver;
+   return desc;
  }
 
  function badPgn(pgn) {
    return pgn == 126993;
  }
+let isOpen = false;
 
+function toggleCollapse() {
```

Comment:
is this used?

Reply:
In listening to Seth he made the case that its too busy with all open.  All closed lets you drill down into only the NMEA instruments of interest.